src/main/java/Event.java

```diff
@@ -0,0 +1,15 @@
+public class Event extends Task {
+    protected String from;
+    protected  String to;
```

Comment:
Just a suggestion but maybe you could rename these to "startDate" "endDate" so it's clearer :)